.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,38 @@
 
+---
+**PR checklist**
+
+The following checklist shall help the PR's author, the reviewers and maintainers to ensure the quality of this project.
+It is based on our contributors guidelines, especially the ["writing code" section](https://github.com/junit-pioneer/junit-pioneer/blob/master/CONTRIBUTING.md#writing-code).
+It shall help to check for completion of the listed points.
+If a point does not apply to the PR changes, the entry can just be marked as done. 
+
+Documentation (general)
+ 
+* [ ] There is documentation (Javadoc and site documentation; added or updated)
+* [ ] There is implementation information to describe _why_ a non-obvious source code / solution got implemented
+* [ ] Site documentation has its own `.adoc` file in the `docs` folder, e.g. `docs/report-entries.adoc`
+* [ ] Only one sentence per line (especially in `.adoc` files)
+* [ ] Javadoc uses formal style, while sites documentation may use informal style (see #265)
+
+Documentation (new extension)
+* [ ] The `docs/docs-nav.yml` navigation has an entry for the new extension
+* [ ] The `package-info.java` contains information about the new extension
+
+Code
+* [ ] Code satisfies code style, naming conventions etc.
+* [ ] Successful tests cover all changes
+* [ ] There are checks which validate correct / false usage / configuration of a functionality
+There are tests to verify those checks (see #164)
+* [ ] Tests use [AssertJ](https://joel-costigliola.github.io/assertj/) or our own PioneerAssertions (which are based on AssertJ)
```

Comment:
There is no `PioneerAssertions` class but `PioneerAssert`? We may also point to the corresponding package, a good example or the anchor in `CONTRIBUTING.md`?